sql_generators/mobile_kpi_support_metrics/templates/new_profile_clients.bigconfig.yml

```
@@ -21,6 +21,23 @@ tag_deployments:
           - saved_metric_id: is_99_percent_valid_normalized_channel
             rct_overrides:
               - first_seen_date
+      - column_selectors:
+        - name: {{ project_id }}.{{ project_id }}.{{ dataset }}_derived.{{ name }}_{{ version }}.*
```

```suggestion
        - name: {{ project_id }}.{{ dataset }}_derived.{{ name }}_{{ version }}.*
```

should project_id be twice?

---

oh, i see that it is above